interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,430 @@ components:
+        This state must transition to Ended.  In this state, the
+        `/uss/v1/operational_intents/{entityid}/telemetry` USS-USS endpoint
+        should respond, if available, to queries from USS peers.  The create or
+        update request for this operational intent may omit a Key in this case
+        because the operational intent is being adjusted as flown and cannot
+        necessarily deconflict.
+      type: string
+      enum:
+      - Accepted
+      - Activated
+      - Nonconforming
+      - Contingent
+    Key:
+      description: >-
+        Proof that a client has obtained the latest airspace content, used to
+        ensure that write operations to the DSS occur only when the latest
+        content is known (i.e. has been read). The client is required to pass a
+        current Key constructed from information obtained with previous read
+        operations and subsequent requests for full information, and optionally,
+        subscription updates for deconflicted write operations to the DSS.  The
+        contents of this data structure are generated by the client.
+      type: array
+      items:
+        $ref: '#/components/schemas/EntityOVN'
+      default: [ ]
+    SubscriptionUssBaseURL:
+      description: >-
+        The base URL of a USS implementation of the parts of the USS-USS API
+        necessary for receiving the notifications requested by this
+        subscription.
+      anyOf:
+        - $ref: "#/components/schemas/UssBaseURL"
+
+    ImplicitSubscriptionParameters:
+      description: >-
+        Information necessary to create a subscription to serve a single
+        operational intent's notification needs.
+      type: object
+      required:
+      - uss_base_url
+      properties:
+        uss_base_url:
+          description: >-
+            The base URL of a USS implementation of the parts of the USS-USS API
+            necessary for receiving the notifications that the operational
+            intent must be aware of.  This includes, at least, notifications for
+            relevant changes in operational intents.
+          anyOf:
+          - $ref: '#/components/schemas/SubscriptionUssBaseURL'
+        notify_for_constraints:
+          description: >-
+            True if this operational intent's subscription should trigger
+            notifications when constraints change. Otherwise, changes in
+            constraints should not trigger notifications.  The scope
+            utm.constraint_processing is required to set this flag true, and a
+            USS performing the constraint processing role should set this flag
+            true.
+          type: boolean
+          default: false
+          
+    #########
+    # The definitions before this seperator are from the standard UTM API (https://github.com/astm-utm/Protocol/blob/master/utm.yaml), the definitions specific for the automated testing suite follow below.
+    #########
+    
+    OperationalIntentTestInjection:
```

Comment:
We need to provide [the information a USS needs in order to PUT a new operational intent](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/astm-utm/Protocol/master/utm.yaml#operation/createOperationalIntentReference) which they cannot obtain for themselves.

Fields needed:
* `extents`
* `state`

Fields generated/obtained/provided by USS, and therefore not needed (or appropriate) in this data structure:
* `key` (this is proof of detecting conflicts so therefore must be provided by the USS)
* `uss_base_url` (this depends on the USS's hosting infrastructure and cannot be known by uss_qualifier)
* `subscription_id` (this is the USS's subscription which will keep them informed about airspace changes and cannot be known by uss_qualifier)
* `new_subscription` (this is a request to create a new subscription to keep the USS informed about airspace changes and must be supplied by the USS when presented to the DSS)

After the operational intent reference is created in the DSS, other USSs may contact the USS for [the details](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/astm-utm/Protocol/master/utm.yaml#operation/getOperationalIntentDetails) of that operational intent and the USS must be able to respond.  The additional information it will need in order to respond (beyond that required by the DSS when creating a reference) is:
* `priority`
* `volumes`
* `off_nominal_volumes`

Note that `volumes` is not exactly the same as `extents` -- `extents` must bound `volumes` + `off_nominal_volumes`, but it does not have to equal them.  And, since we need to provide the USS with both `volumes` and `off_nominal_volumes`, there is no need to provide `extents` since those can (and should) be computed from `volumes` + `off_nominal_volumes` by the USS.

Reply:
OK I have now:

Reply:
Thank you for this, I have now: 
- removed `extents`
- removed `key`
- removed `uss_base_url`
- removed `subscription_id`
- removed `new_subscription`
- Updated description to have a note regarding some fields need to be computed.